src/widgets/_lists.scss

```diff
@@ -0,0 +1,23 @@
+list {
+    background-color: bg-color(1);
+
+    popover & {
+        background-color: transparent;
+    }
+
+    row {
+        &:selected {
+            background-color: rgba($fg-color, 0.2);
+
+            &:focus {
+                background-color: #{'@selected_bg_color'};
+                color: #{'@selected_fg_color'};
+            }
+        }
```

Comment:
```suggestion
    row {
        &:selected {
            // Effectively desaturate the background
            background-blend-mode: hue;
            background-color: #{'@selected_bg_color'};
            background-image: linear-gradient(white, white);

            &:focus {
                background-color: #{'@selected_bg_color'};
                background-image: none;
                color: #{'@selected_fg_color'};
            }
        }
```

Reply:
This is much worse for some colors:

![Screenshot from 2020-05-26 12 44 28@2x](https://user-images.githubusercontent.com/7277719/82943729-ec5ced80-9f4e-11ea-95b7-7fa2ec7c1e9a.png)
![Screenshot from 2020-05-26 12 44 38@2x](https://user-images.githubusercontent.com/7277719/82943733-ecf58400-9f4e-11ea-97f0-cb81c701611c.png)
![Screenshot from 2020-05-26 12 44 46@2x](https://user-images.githubusercontent.com/7277719/82943738-ed8e1a80-9f4e-11ea-9fe6-d85aa4ec2b79.png)